From a21445e557042def4c868b296d64a11e91d19c6a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:27:06 +0800
Subject: [PATCH] Revert "cpufreq: ondemand: Don't synchronize sample rate
 unless mulitple cpus present"

This reverts commit beb070f3d41fb82c69c6dcec1293817c7c788d10.
---
 drivers/cpufreq/cpufreq_ondemand.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/cpufreq/cpufreq_ondemand.c b/drivers/cpufreq/cpufreq_ondemand.c
index 1fdbbe4d2e3d..8fb07103628c 100644
--- a/drivers/cpufreq/cpufreq_ondemand.c
+++ b/drivers/cpufreq/cpufreq_ondemand.c
@@ -585,9 +585,7 @@ static void do_dbs_timer(struct work_struct *work)
 	/* We want all CPUs to do sampling nearly on same jiffy */
 	int delay = usecs_to_jiffies(dbs_tuners_ins.sampling_rate);
 
-	if (num_online_cpus() > 1)
-		delay -= jiffies % delay;
-
+	delay -= jiffies % delay;
 	mutex_lock(&dbs_info->timer_mutex);
 
 	/* Common NORMAL_SAMPLE setup */
-- 
2.35.3

